Formula/fontforge.rb

```diff
@@ -13,18 +12,21 @@ class Fontforge < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "python" => [:build, :test]
   depends_on "cairo"
   depends_on "fontconfig"
+  depends_on "freetype"
   depends_on "gettext"
   depends_on "giflib"
+  depends_on "glib"
   depends_on "jpeg"
   depends_on "libpng"
   depends_on "libspiro"
   depends_on "libtiff"
   depends_on "libtool"
   depends_on "libuninameslist"
   depends_on "pango"
+  depends_on "python"
```

Comment:
Otherwise `brew audit` will complain

```
fontforge:
  * C: 16: col 3: dependency "cairo" (line 16) should be put before dependency "python" (line 15)
  * C: 17: col 3: dependency "fontconfig" (line 17) should be put before dependency "python" (line 15)
  * C: 18: col 3: dependency "freetype" (line 18) should be put before dependency "python" (line 15)
  * C: 19: col 3: dependency "gettext" (line 19) should be put before dependency "python" (line 15)
  * C: 20: col 3: dependency "giflib" (line 20) should be put before dependency "python" (line 15)
  * C: 21: col 3: dependency "glib" (line 21) should be put before dependency "python" (line 15)
  * C: 22: col 3: dependency "jpeg" (line 22) should be put before dependency "python" (line 15)
  * C: 23: col 3: dependency "libpng" (line 23) should be put before dependency "python" (line 15)
  * C: 24: col 3: dependency "libspiro" (line 24) should be put before dependency "python" (line 15)
  * C: 25: col 3: dependency "libtiff" (line 25) should be put before dependency "python" (line 15)
  * C: 26: col 3: dependency "libtool" (line 26) should be put before dependency "python" (line 15)
  * C: 27: col 3: dependency "libuninameslist" (line 27) should be put before dependency "python" (line 15)
  * C: 28: col 3: dependency "pango" (line 28) should be put before dependency "python" (line 15)
```

😜